### PR TITLE
[tutorial] your first fungible asset

### PR DESCRIPTION
**Pull request by lightmark**
### Description
similar to your first coin. but with FA.

### Test Plan
pnpm start

### CODE REVIEW

File: aptos-move/move-examples/fungible_asset/managed_fungible_asset/Move.toml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [package]
-name = "ManagedFungibleAsset"
+name = "MoonCoin"
```
Comment (movekevin) [inline aptos-move/move-examples/fungible_asset/managed_fungible_asset/Move.toml:2]: Lets not name new things like this 😄

File: aptos-move/move-examples/fungible_asset/moon_coin/sources/MoonCoin.move [hunk 2]
```diff
@@ -80,20 +82,12 @@ module example_addr::simple_managed_coin {
...
-        let asset = get_metadata();
-        let transfer_ref = &authorized_borrow_refs(admin, asset).transfer_ref;
-        let wallet = primary_fungible_store::ensure_primary_store_exists(account, asset);
-        fungible_asset::set_frozen_flag(transfer_ref, wallet, true);
-    }
-
-    /// Unfreeze an account so it can transfer or receive fungible assets.
-    public entry fun unfreeze_account(admin: &signer, account: address) acquires ManagedFungibleAsset {
+    /// Freeze/Unfreeze an account so it cannot transfer or receive fungible assets.
+    public entry fun set_frozen_flag(admin: &signer, account: address, frozen: bool) acquires ManagedFungibleAsset {
```
Comment (movekevin) [inline aptos-move/move-examples/fungible_asset/moon_coin/sources/MoonCoin.move:86]: I actually like having separate freeze and unfreeze functions for more expressiveness

File: developer-docs-site/docs/tutorials/first-fungible-asset.md [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+import TabItem from '@theme/TabItem';
+
+# Your First Fungible Asset
+
+This tutorial introduces how you can compile, deploy, and mint your own fungible asset (FA), named [Moon Coin](https://github.com/aptos-labs/aptos-core/tree/main/aptos-move/move-examples/fungible_asset/moon_coin).
+Make sure you have understood FA before moving on to the tutorial. If not, it is highly recommended to read it first.
+
+* [Fungible Asset](../standards/fungible-asset.md)
+
+## Step 1: Pick an SDK
```
Comment (movekevin) [inline developer-docs-site/docs/tutorials/first-fungible-asset.md:16]: I wonder if we can point to a common section for setting up sdk and cli and cloning aptos-core as all tutorials need those same steps. Cc @saharct

File: aptos-move/move-examples/fungible_asset/moon_coin/sources/MoonCoin.move (hunk 2 above)
Comment (lightmark) [reply aptos-move/move-examples/fungible_asset/moon_coin/sources/MoonCoin.move:86]: that's small nits. I could fix it

**Comment by lightmark** [comment]
@movekevin @davidiw @saharct @xbtmatt

**Review by movekevin — APPROVED** [review]

File: developer-docs-site/docs/tutorials/first-fungible-asset.md [hunk 4]
```diff
@@ -0,0 +1,247 @@
...
+### Step 3.2: Completing the example
+
+Returning to the previous prompt, press ENTER as the package is now ready to be published.
+
+The application will complete, printing:
+
+```bash
+Publishing FACoin package.
+
+All the balances in this exmaple refer to balance in primary fungible stores of each account.
```
Comment (xbtmatt) [inline developer-docs-site/docs/tutorials/first-fungible-asset.md:114]: small typo with `exmaple`

File: aptos-move/move-examples/fungible_asset/fa_coin/sources/FACoin.move [hunk 5]
```diff
@@ -1,7 +1,7 @@
 /// A 2-in-1 module that combines managed_fungible_asset and coin_example into one module that when deployed, the
 /// deployer will be creating a new managed fungible asset with the hardcoded supply config, name, symbol, and decimals.
 /// The address of the asset can be obtained via get_metadata(). As a simple version, it only deal with primary stores.
```
Comment (xbtmatt) [inline aptos-move/move-examples/fungible_asset/fa_coin/sources/FACoin.move:3]: ` it only deal with primary stores.` => ` it only deals with primary stores.`

sorry for all the separate typo fixes, think this is the last one!

File: developer-docs-site/docs/tutorials/first-fungible-asset.md [hunk 6]
```diff
@@ -0,0 +1,247 @@
...
+:::tip
+The entity that creates FACoin gains the capabilities for minting, burning, freezing/unfreezing, and forceful transferring between any fungible stores no matter they are frozen or not.
+So `Withdraw`, `Deposit`, and `Transfer` in the management module have different semantics than those described in fungible asset framework that limited by frozen status.
+:::
+
+---
+
+#### Step 4.3.1: Initializing "FACoin" metadata object
+
+After publish the module to the Aptos blockchain, the entity that published that coin type should initialize a metadata object describing the information about this FA:
```
Comment (xbtmatt) [inline developer-docs-site/docs/tutorials/first-fungible-asset.md:196]: `After publish the module` => `After publishing the module`

😄

**Review by xbtmatt — APPROVED** [review]

**Comment by lightmark** [comment]
oh, I forget to disable the auto commit.

**Pull request merged**